core/src/view/view.h

```diff
@@ -258,7 +262,9 @@ class View {
 
     bool m_dirtyMatrices;
     bool m_dirtyTiles;
+    bool m_dirtyConstraint;
     bool m_changed;
+    bool m_constrainToWorldBounds = true;
```

Comment:
Probably makes sense to put this in a bitset, will add another commit.

Reply:
Not worth the complexity - we only have one instance of `View`

Reply:
I meant all the bools now, as View States or something, to save "memory". But again tiny saving, will ignore for now.